### PR TITLE
`tf.nn.softmax` should be calculated in the dimension of `axis=1`

### PR DESCRIPTION
**Pull request by wangzhe258369**

### CODE REVIEW

**Comment by dongjun-Lee** [comment]
@wangzhe258369 Thanks for your pr!
